src/Routes/SamplePage/SamplePage.js

```diff
@@ -1,15 +1,11 @@
 import React from 'react';
-import { withRouter, Link } from 'react-router-dom';
-import { useDispatch } from 'react-redux';
-
-import { Button, StackItem, Stack, Title } from '@patternfly/react-core';
-import { Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
-import { addNotification } from '@redhat-cloud-services/frontend-components-notifications/';
-
-import asyncComponent from '../../Utilities/asyncComponent';
-const SampleComponent = asyncComponent(() => import('../../Components/SampleComponent/sample-component'));
-
+import { withRouter } from 'react-router-dom';
+import { Button, Card, CardBody } from '@patternfly/react-core';
+import { Section, Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
 import './sample-page.scss';
+import { } from '@patternfly/react-core';
+
+const INVENTRY_API_ROOT = '/api/inventory/v1';
```

Comment:
As of now it is the only one constant that I declared. Eventually I need to move this into `src/constants.js`.